ble-events-handler.js

```diff
@@ -0,0 +1,315 @@
+/**
```

Comment:
What if we explain the dependency of this file to the event emitters? And provide some additional description to the format of events?

Reply:
Description and manifest are updated.